frontend/src/index.pug

```diff
@@ -34,6 +34,14 @@ html
     script(src="https://cdn.jsdelivr.net/gh/highlightjs/cdn-release@9.14/build/languages/yaml.min.js")
   body
     #app
+      loading-overlay.overlay-loader(
```

Comment:
Can you try to wrap it with a template?  Otherwise, an unpleasant message will show when the network connection is slow.

![image](https://user-images.githubusercontent.com/35621726/102733880-646e8300-4379-11eb-9036-ffe43730e2fd.png)

Reply:
Fixed with v-cloak